backend/src/main/java/co/inventorsoft/scripty/model/entity/Picture.java

```diff
@@ -0,0 +1,34 @@
+package co.inventorsoft.scripty.model.entity;
+
+import jdk.nashorn.internal.objects.annotations.Constructor;
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+/**
+ * @author A1lexen
+ */
+
+@Entity
+@Getter
+@Setter
+@Table(name="images")
+@RequiredArgsConstructor
+@NoArgsConstructor
+public class Picture {
```

Comment:
Why do we need separate table for images?

Reply:
With such approach it's easier to store picture's extension and then return it